pkg/server/image_test.go

```diff
@@ -0,0 +1,131 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package server
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"strconv"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"k8s.io/kubernetes/pkg/kubelet/api/v1alpha1/runtime"
+)
+
+func cleanupPaths() {
+	for _, p := range []string{
+		"/var/lib/containerd", // TODO use constants when available and if appropriate might want a test dir here
+		"/var/run/containerd",
+	} {
+		os.RemoveAll(p)
+	}
+}
+
+const redisImage = "docker.io/library/redis:latest"
+
+// NOTE: To run the test, please make sure `dist` and `jq` is in $PATH.
```

Comment:
Add `TODO: add fake image service` here,  we will use fake service to do unit test later. WDYT? @mikebrow